packages/shared/src/components/cards/AdCard.tsx

```diff
@@ -43,17 +43,19 @@ export const AdCard = forwardRef(function AdCard(
           <CardImage
             alt="Ad image"
             src={ad.image}
-            className={classNames(
-              'w-full z-1',
-              showBlurredImage && 'inset-0 m-auto absolute',
-            )}
+            className={{
+              image: classNames(
+                'w-full z-1',
+                showBlurredImage && 'm-auto absolute top-full',
```

Comment:
You don't need the inset anymore to span it? (Just double checking)

Reply:
Have reverted this